### ISSUE TITLE
Incoherence version

### ISSUE THREAD

**Issue by iGitScor** [google/material-design-lite#1700]
I've got the latest version of the mdl (v1.0.5) but the style sources are the same as the previous release 

```css

/**
 * material-design-lite - Material Design Components in CSS, JS and HTML
 * @version v1.0.4
 * @license Apache-2.0
 * @copyright 2015 Google, Inc.
 * @link https://github.com/google/material-design-lite
 */

```

Here the package.json

```json

{
  "name": "material-design-lite",
  "version": "1.0.5",
  ...
}

```

**Comment by surma** [comment]
Confirmed for npm and bower, ZIP download is fine. The version in the comment has not been updated for some reason. The code however ist the 1.0.5 code. 

I think we will hold off of pushing a hotfix for this as they tend to introduce more problems, but we will make sure we get it right on the next release. Thanks for issuing, @iGitScor!

cc @addyosmani @sgomes

**Comment by iGitScor** [comment]
The problem is that in the grid style, there is not the offset mixin

**Comment by sgomes** [comment]
@iGitScor The offsets are a new feature in the upcoming v1.1, and are not available in v1.0.5 or any other v1.0.x release.

**Comment by sgomes** [comment]
@surma Have we confirmed that the code is actually old, and this isn't just a labelling problem on the file (e.g. because `gulp all` was run before the version bump)?

**Comment by surma** [comment]
As I said, the code is the v1.0.5 code (empty diff against ZIP download). It merely the wrong version string in the header.

**Comment by addyosmani** [comment]
It looks like this is a labelling problem. We should be able to retag the release without it causing issues but npm will have this problem until the next release gets cut.

**Comment by iGitScor** [comment]
The confusion comes from the grid.md documentation file from the master branch, sorry.
Is it possible to get the v1.1 version with bower ?Is it beta release or dev release ?

Thanks

**Comment by sgomes** [comment]
@surma Oops, sorry, missed that bit!

**Comment by sgomes** [comment]
@iGitScor Sorry, we don't have master exposed via bower :(

If you really want to live on the edge (**for development only, make sure you don't do it in production!**), you could add our repo as a manual dependency yourself, e.g.:

`bower install --save https://github.com/google/material-design-lite.git#master`

Also, we try to keep things running smoothly in master, but you should expect the occasional breakage :)

**Comment by gertcuykens** [comment]
I am still confused, master is 1.0.5 right?
https://github.com/google/material-design-lite/blob/master/bower.json
https://github.com/google/material-design-lite/blob/master/package.json

**Comment by surma** [comment]
@gertcuykens No, master is going towards 1.1. All the bugfix-releases named `1.0.x` are on the `mdl-1.0` branch:

https://github.com/google/material-design-lite/blob/mdl-1.0/bower.json
https://github.com/google/material-design-lite/blob/mdl-1.0/package.json

We haven’t merged `mdl-1.0` into master for a while which is why master is still saying `1.0.4`.

**Comment by gertcuykens** [comment]
Ok thanks, I am only interested in the latest and greatest so if master is 1.1 then master I will be using :)
